_includes/manuals/1.24/5.7.7_openid_sso.md

```diff
@@ -0,0 +1,106 @@
+### Single Sign-on for Foreman using OpenID Connect protocol
+
+An OpenID Provider(OP)  implements a Single Sign-on (SSO) using  an OpenID Connect (OIDC) protocol. OIDC verifies the identity of the End-user depending on the given authentication details. In this example, [Keycloak](https://www.keycloak.org/) is used as the OpenID provider, and Foreman is the Relying Party (RP)
+
+OpenID provider provides an ID Token, encoded in JSON Web Token (JWT) to Foreman.
+
+An example of an encoded JWT ID Token:
+
+```
+eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyfQ.SflKxwRJSMeKKF2QT4fwpMeJf36POk6yJV_adQssw5c
+```
+
+#### Configuring Foreman
+
+As a system administrator, execute the following procedure to configure Foreman.
+
+Prerequisites
+
+* A working installation of Foreman at https://foreman.example.com.
+* A working installation of OpenID provider, for example, Keycloak at https://sso.example.com, which uses OIDC protocol. Ensure that the Keycloak server is running.
+* Ensure that both RP and OpenID provider are using https instead of http. If the certificates or the CA is self-signed, ensure that they are added to the End-user certificate trust store.
+
+Procedure
+
+1. Install the mod_auth packages:
+```
+# yum install mod_auth_openidc python2-requests-oauthlib keycloak-httpd-client-install -y
```

Comment:
At least on CentOS 7.7 `python2-requests-oauthlib` can be dropped because it's pulled in automatically.

Reply:
@ekohl good catch! I checked for Fedora, it comes as an installing dependency for the `keycloak-httpd-client-install` package.

Reply:
Could you have a look at this?
```suggestion
# yum install mod_auth_openidc keycloak-httpd-client-install -y
```

Reply:
@ekohl , I have updated this command.